contribs/gmf/src/d3-lidar-profile/lidarprofilePlot.js

```diff
@@ -0,0 +1,320 @@
+goog.provide('gmf.lidarProfile.plot');
+
+/**
+* @constructor
+* @param {Object} options to be defined in gmfx
+* @param {gmf.lidarProfile} parent instance
+*/
+gmf.lidarProfile.plot = function(options, parent) {
+
+/**
+ * @type {Object}
+ * @export
+ */
+  this.options = options;
+
+  /**
+  * @type {gmf.lidarProfile}
+  * @private
+  */
+  this.parent_ = parent;
+
+  /**
+  * @type {gmf.lidarProfile.utils}
+  * @private
+  */
+  this.utils_ = new gmf.lidarProfile.utils(options, null);
+
+};
+
+/**
+ * Draw the points to the canvas element
+ * @param {gmfx.LidarProfilePoints} points of the profile
+ * @param {string} material used to determine point color
+ * @export
+*/
+gmf.lidarProfile.plot.prototype.drawPoints = function(points, material) {
+  let i = -1;
+  const n = points.distance.length;
```

Comment:
Please, use human-readable name for variables (except for `i`)

Reply:
Fixed

Reply:
Fixed